### ISSUE TITLE
Manifest error with firefox 63

### ISSUE THREAD

**Issue by char101** [piroor/treestyletab#2066]
## Short description

```
Reading manifest: Error processing permissions.4: Value "menus.overrideContext" must either: must either [must either [be one of ["clipboardRead", "clipboardWrite", "geolocation", "idle", "notifications"], be one of ["bookmarks"], be one of ["find"], be one of ["history"], be one of ["search"], be one of ["activeTab", "tabs", "tabHide"], be one of ["browserSettings"], be one of ["cookies"], be one of ["downloads", "downloads.open"], be one of ["topSites"], be one of ["webNavigation"], or be one of ["webRequest", "webRequestBlocking"]], be one of ["alarms", "mozillaAddons", "storage", "unlimitedStorage"], be one of ["browsingData"], be one of ["devtools"], be one of ["identity"], be one of ["menus", "contextMenus"], be one of ["pkcs11"], be one of ["geckoProfiler"], be one of ["sessions"], be one of ["contextualIdentities"], be one of ["dns"], be one of ["management"], be one of ["privacy"], be one of ["proxy"], be one of ["nativeMessaging"], be one of ["telemetry"], be one of ["theme"], or match the pattern /^experiments(\.\w+)+$/], or must either [be one of ["<all_urls>"], must either [match the pattern /^(https?|wss?|file|ftp|\*):\/\/(\*|\*\.[^*/]+|[^*/]+)\/.*$/, or match the pattern /^file:\/\/\/.*$/], or match the pattern /^resource:\/\/(\*|\*\.[^*/]+|[^*/]+)\/.*$|^about:/]
Reading manifest: Error processing variable_color_icons: An unexpected property was found in the WebExtension manifest.
```

## Steps to reproduce

 1. Enable add-on debugging
 2. View the row for treestyletab

## Environment

 * Platform (OS): Windows 10
 * Version of Firefox: 63
 * Version (or revision) of Tree Style Tab: 2.6.4

**Comment by piroor** [comment]
Those "errors" are intentional.

 * `menus.overrideContext` is required for [advanced context menu on Firefox 64 and later](https://piro.sakura.ne.jp/latest/blosxom/mozilla/xul/2018-10-14_override-context-on-fx64.htm#topic2018-10-14_override-context-on-fx64). I have less resource to provide multiple packages "for Firefox 63 or older" and "for Firefox 64 and later" separately, so I decided to keep the error there until addon linter reject such an error in the future.
 * I decided to put definitions for optional toolbar icons as `variable_color_icons` in the manifest.json, because I'll forget that if I put such an information in any different place. I'm planning to unify SVG icon definitions to which one after the [bug 1391980](https://bugzilla.mozilla.org/show_bug.cgi?id=1391980) or [1367042](https://bugzilla.mozilla.org/show_bug.cgi?id=1367042) become fixed. Same to above, I keep the error there until addon linter reject such an error in the future.
